src/Bot.js

```diff
@@ -5,11 +5,23 @@ const { OMG } = require("./OMG")
 const bot = new Telegraf(process.env.BOT_TOKEN || "")
 const Omg = new OMG()
 
+const veryFunnyMessagesAboutFlutter = [
+  "🚨<a href='https://i.imgur.com/Sml2Ayv.png'>&#8205;</a>",
+  "🚨<a href='https://i.imgur.com/m0gb4Qa.png'>&#8205;</a>",
+  "🚨<a href='https://i.imgur.com/SnSjGMa.png'>&#8205;</a>",
+  "🚨<a href='https://i.imgur.com/GsdpgXF.png'>&#8205;</a>",
+  "🚨<a href='https://i.imgur.com/ilfzQUW.png'>&#8205;</a>",
+  "🚨<a href='https://i.imgur.com/mlwDtRi.png'>&#8205;</a>",
+  "Finally we know: HTML and regex<a href='https://i.imgur.com/10nBaqX.png'>&#8205;</a>",
+  "<i>beep boop</i> flutter è una 💩<a href='https://www.reddit.com/r/mAndroidDev/comments/kp6ks7/now_tell_me_which_one_of_you_did_this/?utm_source=share&utm_medium=web2x&context=3)'>.</a>",
+  "<i><a href='https://medium.com/@acedened/ios-app-from-flutters-showcase-page-might-not-use-flutter-at-all-23488ff82407'>Flutter is so great that mostly native app is showcased on Flutter page</a>"
+]
+
```

Comment:
Magari in futuro facciamo un file di costanti

Reply:
Ci sta